cmd/fleet/serve.go

```diff
@@ -280,7 +280,11 @@ the way that the Fleet server works.
 				RedisScanKeysCount: config.Osquery.AsyncHostRedisScanKeysCount,
 				CollectorInterval:  config.Osquery.AsyncHostCollectInterval,
 			}
-			svc, err := service.NewService(ds, task, resultStore, logger, osqueryLogger, config, mailService, clock.C, ssoSessionStore, liveQueryStore, carveStore, *license, failingPolicySet)
+
+			// TODO: gather all the different contexts and use just one
```

Comment:
Can you make an issue for this please?

Reply:
https://github.com/fleetdm/fleet/issues/3750